circle.yml

```diff
@@ -0,0 +1,19 @@
+general:
+    artifacts:
+      - .build_ano/uno/firmware.elf
+      - .build_ano/uno/firmware.hex
+
+dependencies:
+    pre:
+        - sudo apt-get install arduino arduino-core arduino-mk
+        - sudo rm -rf /usr/share/arduino/libraries/Robot_Control
+        - git clone https://github.com/sourcebots/Arturo && cd Arturo && sudo make install
```

Comment:
We could, but seeing as this is tooling rather than a build dependency, it seems better installed as one